dask/dataframe/io/parquet/core.py

```diff
@@ -64,6 +65,12 @@ def __init__(
         self.kwargs = kwargs
         self.part_ids = list(range(len(parts))) if part_ids is None else part_ids
 
+        # `kwargs` = user-defined kwargs to be passed for all parts
+        self.kwargs = kwargs
```

Comment:
It looks like we set `self.kwargs = kwargs` a few lines above. Perhaps we should move this comment up there

Reply:
Oops - I'll actually just remove the redundant def above.